Minor bugfixes and docfixes (#805)
Deep-MI/FastSurfer: README.md

   - outputs a hypothalamic subsegmentation including 3rd ventricle, c. mammilare, fornix and optic tracts.
   - a T1w image is highly recommended ([notes on input images](#requirements-to-input-images)), supports high-res (up to 0.7mm, but experimental beyond that).
   - allows the additional passing of a T2w image with `--t2 <path>`, which will be registered to the T1w image (see `--reg_mode` option).

>    - calculates summary statistics based on the T1w image; biasfield-corrected inputs are recommended for best results.

Reviewer: I am not really sure what our terminology for those files should be...
Also, rather than fixing the documentation, we should probably fix the code here. The intention is the other way around: Only compute partial volume stats, if a biasfield corrected image is available.

```suggestion
   - calculates summary statistics based on the biasfield-corrected T1w image  (skipped if `--no_biasfield` is passed).
```
